### PR TITLE
wasm-objdump: Add function offset when disassembling

### PR DESCRIPTION
**Pull request by sbc100**
Currently we show file offset in hex, but backtraces in v8 show
functions offset in decimal

This change adds the decimal offset in addition.  Perhaps we could
drop the hex file offset?  Or make it an option?

### CODE REVIEW

File: src/binary-reader-objdump.cc [hunk 1]
```diff
@@ -643,7 +646,8 @@ Result BinaryReaderObjdumpDisassemble::OnEndExpr() {
 
 Result BinaryReaderObjdumpDisassemble::BeginFunctionBody(Index index,
                                                          Offset size) {
-  printf("%06" PRIzx " func[%" PRIindex "]", state->offset, index);
+  current_function_offset = state->offset;
+  printf("%06" PRIzx " func[%02" PRIindex "]", state->offset, index);
```
Comment (binji) [inline src/binary-reader-objdump.cc]: Seems a bit strange to me to use a leading zero for a decimal number. Also, none of the other indexes have leading zeroes.

Comment (sbc100) [reply src/binary-reader-objdump.cc]: Its for alignment of the instructions which are on the right of this information.

**Comment by sbc100** [comment]
So this a v8 bug I guess.

**Comment by sbc100** [comment]
Opened a v8 issue: https://bugs.chromium.org/p/v8/issues/detail?id=9172

**Pull request closed without merging**